src/components/BrowseResultsList/getBrowseResultsFormatter.js

```diff
@@ -0,0 +1,46 @@
+import MissedMatchItem from '../MissedMatchItem';
+
+const getFullMatchRecord = (item, isAnchor) => {
+  if (isAnchor) {
+    return <strong>{item}</strong>;
+  }
+
+  return item;
+};
+
+const getBrowseResultsFormatter = (data) => {
+  return {
+    'title': r => getFullMatchRecord(r.instance?.title, r.isAnchor),
+    'subject': r => {
+      if (r?.totalRecords) {
+        return getFullMatchRecord(r?.subject, r.isAnchor);
+      }
+      return <MissedMatchItem query={r.subject} />;
+    },
+    'callNumber': r => {
+      if (r?.instance || r?.totalRecords) {
+        return getFullMatchRecord(r?.fullCallNumber, r.isAnchor);
+      }
+      return <MissedMatchItem query={r.fullCallNumber} />;
+    },
+    'contributor': r => {
+      if (r?.totalRecords) {
+        return getFullMatchRecord(r?.name, r.isAnchor);
+      }
+      return <MissedMatchItem query={r.name} />;
```

Comment:
Contributor cell has already been updated [here](https://github.com/folio-org/ui-inventory/commit/cf25f8ebce1fe18299cf158eaf96fa3a3c2a4998).

Reply:
Thanks @Dmytro-Melnyshyn, but looks like these are fresh changes and were not part of the existing implementation. This PR is not final for browse refactoring, so this and other differences could be updated in next PRs.